test/torchaudio_unittest/models/tacotron2/model_test.py

```diff
@@ -0,0 +1,202 @@
+
+import torch
+from torchaudio.models.tacotron2 import _Tacotron2, _Encoder, _Decoder
+from torchaudio_unittest.common_utils import (
+    TorchaudioTestCase,
+    skipIfNoCuda,
+    TempDirMixin,
+)
+from parameterized import parameterized
+
+
+class TorchscriptConsistencyMixin(TempDirMixin):
+    """Mixin to provide easy access assert torchscript consistency"""
+
+    def _assert_torchscript_consistency(self, model, tensors):
+        path = self.get_temp_path('func.zip')
+        torch.jit.script(model).save(path)
+        ts_func = torch.jit.load(path)
+
+        torch.random.manual_seed(40)
+        output = model(*tensors)
+
+        torch.random.manual_seed(40)
+        ts_output = ts_func(*tensors)
+
+        self.assertEqual(ts_output, output)
+
+
+class TestEncoder(TorchaudioTestCase, TorchscriptConsistencyMixin):
+
+    def _get_model(self, encoder_n_convolutions=3, encoder_embedding_dim=512, encoder_kernel_size=5):
```

Comment:
There seems to be no benefit of making this into a method. How about calling `_Encoder` directly?

Reply:
Fixed in the new commit.
Thanks.